packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx

```diff
@@ -57,15 +57,17 @@ describe('<ElementAvatar>', () => {
       <ElementAvatar
         userId="@user-alice"
         displayName="Alice"
-        avatarUrl={'mxc://alice'}
+        avatarUrl={'mxc://example.com/imageAlice'}
       />,
     );
 
     await expect(
       screen.findByRole('presentation', { hidden: true }),
     ).resolves.toHaveAttribute(
       'src',
-      expect.stringMatching(/\/_matrix\/media\/r0\/thumbnail\/alice/i),
+      expect.stringMatching(
+        /https:\/\/.*\/_matrix\/media\/r0\/thumbnail\/example\.com\/.*imageAlice/i,
```

Comment:
> it('should render avatar url'

I'm not sure, should it? Can't avatars be authenticated media? Did we miss this before when implementing auth media?

Reply:
I guess [this method](https://github.com/nordeck/matrix-widget-toolkit/blob/main/packages/mui/src/components/ElementAvatar/createAvatarUrl.ts) should be fully reviewed under that light.

Maybe something to work on another PR / ticket?

Reply:
this PR only to fixed the MXC URLs in the  tests and not the implementation. If necessary,  if necessary we should open a ticket.

Reply:
I've opened it internally. We'll tackle this comment there.